osf/management/commands/fetch_cedar_metadata_templates.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+
+from osf.models import CedarMetadataTemplate
+from osf.external.cedar.client import CedarClient
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        ids = CedarClient().retrieve_all_template_ids()
+        for cedar_id in ids:
+            json = CedarClient().retrieve_template_by_id(cedar_id)
+            title = json['title']
+            older_versions = CedarMetadataTemplate.objects.filter(cedar_id=cedar_id).order_by('-template_version')
+            latest_version = older_versions.first()
+            if latest_version and json != latest_version.template:
```

Comment:
One change I am adding to this is to use `"pav:lastUpdatedOn": "2023-12-08T07:04:58-08:00",` as version tracker since the json is quite large. Whenever we detect the json change, `pav:lastUpdatedOn` is guaranteed to be changed.